app/Main.hs

```diff
@@ -536,6 +541,11 @@ addFromBower name = do
   where
     stripBowerNamePrefix s = fromMaybe s $ T.stripPrefix "purescript-" s
     mkPackageName' = Bifunctor.first show . mkPackageName . stripBowerNamePrefix
+    unpackSpecifier = Bifunctor.second (fmap T.tail . textToMaybe) . T.breakOn "#"
+
+textToMaybe :: Text -> Maybe Text
```

Comment:
Sorry, forgot to ask, could you put this in the same `where` block as above and name it "getVersion" or something more descriptive to its purpose?

Reply:
Refactored this, what do you think?